src/accordion/accordion.directive.ts

```diff
@@ -0,0 +1,305 @@
+import {
+	AfterContentInit,
+	ChangeDetectorRef,
+	Component,
+	ContentChild,
+	ContentChildren,
+	Directive,
+	EventEmitter,
+	forwardRef,
+	Inject,
+	Input,
+	Output,
+	QueryList,
+	TemplateRef,
+} from '@angular/core';
+import { Subscription } from 'rxjs';
+import { NgbAccordionConfig } from './accordion-config';
+import { NgTemplateOutlet } from '@angular/common';
+import { NgbCollapse } from '../collapse/collapse';
+
+let nextId = 0;
+
+@Component({
+	selector: '[ngbAccordionBody]',
+	standalone: true,
+	imports: [NgTemplateOutlet],
+	host: { '[class.accordion-body]': 'true' },
+	template: ` <ng-template [ngTemplateOutlet]="addToDOM() ? bodyTpl : null" ]></ng-template> `,
+})
+export class NgbAccordionBody {
+	constructor(@Inject(forwardRef(() => NgbAccordionItem)) private item: NgbAccordionItem) {}
+
+	@ContentChild(TemplateRef) bodyTpl: TemplateRef<any>;
+
+	addToDOM() {
+		return this.item.destroyOnHide === false || this.item.animatingBodyCollapse;
```

Comment:
`this.item.destroyOnHide` is only initialized once, so it might be wrong I guess.